.github/stale.yml

```diff
@@ -0,0 +1,29 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 14
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - bug
+  - bug - failure to compile
+  - bug - ice
+  - bug - incorrect behaviour
+  - bug - memory leak
+  - bug - miscompile
+  - bug - regression
+  - bug - segfault
+  - performance - compile time
+  - performance - run time
+  - performance
+  - feature_request
+  - enhancement
+  - task
```

Comment:
Am wondering also if this needs to be focused on issues only, else PRs that take a while get closed? e.g. https://github.com/probot/stale/blame/master/README.md#L68-L78 ?

Reply:
yes, issues only,I'll go  and  fix it up.

Reply:
added and pushed